src/cancergenomeinterpreter/index.js

```diff
@@ -379,14 +380,31 @@ const fetchAbstract = async (conn, abstractString) => {
         throw new Error(`unable to parse abstract from ${abstractString}`);
     }
     const [, source, year,, abstractNumber] = m;
-    const abstract = await conn.getUniqueRecordBy({
-        filters: [
-            { year },
-            { source: { filters: { name: source }, target: 'Source' } },
-            { abstractNumber },
-        ],
-        target: 'Abstract',
-    });
+    let abstract;
+
+    try {
+        abstract = await conn.getUniqueRecordBy({
+            filters: [
+                { year },
+                { source: { filters: { name: source }, target: 'Source' } },
+                { abstractNumber },
+            ],
+            target: 'Abstract',
+        });
+    } catch (err) {
+        if (source !== 'ASCO') {
+            throw err;
+        }
+        const absList = (await _asco.fetchAndLoadByIds(conn, [abstractNumber])).filter(a => a.year === year);
+
+        if (absList.length > 1) {
```

Comment:
can this ever happen with correct input values?

Reply:
Unfortunately yes. They re-use abstract numbers at different meetings in the same year. We might be able to add an assumption to this that they mean the general annual meeting but I can't guarantee that so I am just erroring on ambiguous input at the moment